mesonbuild/scripts/meson_install.py

```diff
@@ -132,30 +132,19 @@ def install_headers(d):
 def run_install_script(d):
     env = {'MESON_SOURCE_ROOT' : d.source_dir,
            'MESON_BUILD_ROOT' : d.build_dir,
-           'MESON_INSTALL_PREFIX' : d.prefix
+           'MESON_INSTALL_PREFIX' : d.prefix,
```

Comment:
actually I think we should expose here all LIBDIR/BINDIR and all other directories..